server/src/main/java/io/spine/server/command/model/CommandSubstituteSignature.java

```diff
@@ -37,10 +37,10 @@
 import java.lang.reflect.Method;
 
 /**
- * A signature of {@link io.spine.server.command.model.CommandSubstituteMethod
+ * A signature of {@link CommandSubstituter
  * CommandSubstituteMethod}.
```

Comment:
Please move back a line and update the alt text. Or better, kill it, at it is no longer needed.

I also think it may be a good idea to run a plain search for strings doubling the "old" names of the renamed types. There may be more occurrences like this one.